src/Services/Ordering/Ordering.BackgroundTasks/.editorconfig

```diff
@@ -0,0 +1,132 @@
+###############################
```

Comment:
We should delete these .editorconfig files and just have one in root folder of the repo or in /src